packages/clay-table/demos/a11y.html

```diff
@@ -220,6 +230,63 @@ <h1 class="page-title">
 			showActionsMenu: true,
 			spritemap: spritemap,
 		}, '#container');
+
+		// Table customize
+		new metal.ClayTable({
+			tableClasses: 'table table-autofit table-responsive',
+			useDefaultClasses: false,
+			events: {
+				'cellContentClicked': (event) => {
+					event.data.event.target.closest('tr').classList.toggle('active');
```

Comment:
It think this should work, but here, instead of traversing the dom, we could simply do:

```
this.items = items.map(
    item => Object.assign({}, item, event.data.id == item.id ? {rowClasses: 'table-row active'} : {rowClasses: ''});
);
```

What do you think @matuzalemsteles,  @gcmznt, @FabioDiegoMastrorilli?